docker/Dockerfile

```diff
@@ -0,0 +1,40 @@
+ARG PYTHON_VERSION
+FROM python:${PYTHON_VERSION}-slim
```

Comment:
How about using alpine images? Do you think we will have more issues maintaining them?

Reply:
I'd rather not - developers will be more familiar with the tooling present in the standard images, and size is not that much of a concern. There are also issues with the system packages required, versions etc (especially with geospatial ones).